### PR TITLE
Grudgecodes Nuka Cola into uselessness

### PR DESCRIPTION
**Pull request by Arzion5**
:cl: Salty Traitor
del: Nuka Cola no longer makes you go fast
/:cl:

I died from a person who killed me using the speed from nuka cola, please delet everything I don't like

### CODE REVIEW

**Comment by Blute1** [comment]
lol

File: hippiestation/code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -69,3 +69,6 @@
 			H.hair_style = "Big Afro"
 
 	..()
+
+/datum/reagent/consumable/nuka_cola/on_mob_life(mob/living/M)
+	M.status_flags |=
```
Comment (ghost) [inline hippiestation/code/modules/reagents/chemistry/reagents/drink_reagents.dm]: this doesn't even work

merge right now

**Comment by JohnGinnane** [comment]
Because it isn't abundantly clear what's different, here's the current proc:

image

File: hippiestation/code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 2]
```diff
@@ -69,3 +69,14 @@
...
+/datum/reagent/consumable/nuka_cola/on_mob_life(mob/living/M)
+	M.Jitter(20)
+	M.set_drugginess(30)
+	M.dizziness +=5
+	M.drowsyness = 0
+	M.AdjustSleeping(-40, FALSE)
+	if (M.bodytemperature > 310)//310 is the normal bodytemp. 310.055
+		M.bodytemperature = max(310, M.bodytemperature - (5 * TEMPERATURE_DAMAGE_COEFFICIENT))
+	..()
+. = 1
```
Comment (JohnGinnane) [inline hippiestation/code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Missing indentation

**Review by JohnGinnane — CHANGES_REQUESTED** [review]
Honestly I think you should just remove the line from the original file and put in a comment say `// Hippie - Removed fast because it's OP` or something, rather than copying the entire proc

File: hippiestation/code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 3]
```diff
@@ -69,3 +69,6 @@
 			H.hair_style = "Big Afro"
 
 	..()
+
+/datum/reagent/consumable/nuka_cola/on_mob_life(mob/living/M)
+	M.status.flags |= NONE
```
Comment (JohnGinnane) [inline hippiestation/code/modules/reagents/chemistry/reagents/drink_reagents.dm:74]: Homey this is even worse, it won't do any of the things it should :(

**Review by JohnGinnane — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 3 above)
Comment (JohnGinnane) [reply hippiestation/code/modules/reagents/chemistry/reagents/drink_reagents.dm:74]: I think you should go to here

https://github.com/HippieStation/HippieStation/blob/bb368d910f501154d54129f09d1bf27fd3a7162b/code/modules/reagents/chemistry/reagents/drink_reagents.dm#L389

And replace that line with a commenting saying it was removed or something, and then get ride of the override in the new file altogether

**Comment by ghost** [comment]
broken code and stale

**Pull request closed without merging**